ocaml/xapi/features.ml

```diff
@@ -87,6 +88,7 @@ let keys_of_features =
 		Xen_motion, ("restrict_xen_motion", Negative, "XenMotion");
 		Guest_ip_setting, ("restrict_guest_ip_setting", Negative, "GuestIP");
 		AD, ("restrict_ad", Negative, "AD");
+		Ssl_legacy_switch, ("disable_ssl_legacy_switch", Negative, "Ssl_legacy_switch");
```

Comment:
I just noticed that this has a "disable_" prefix, while the "standard" is "restrict_". This is potentially confusing. Can we please change it?